packages/apps/docs/src/docs/build/pact/beginner/language-basics.md

```diff
@@ -1,205 +1,196 @@
 ---
-title: Explore Pact Language Basics
-description: Kadena makes blockchain work for everyone.
+title: Explore basic functions
+description: Start learning the Pact programming language with an introduction to built-in functions and writing your own simple functions.
 menu: Smart contracts
-label: Explore language basics
+label: Explore basic functions
 order: 4
 layout: full
 tags: ['pact', 'language basics']
 ---
 
-# Explore Pact language basics
+# Explore basic functions
 
-In [Set up a local development network](/build/pact/dev-network), you learned how to set up a local development environment and connect to it using Chainweaver. As part of the introduction to the development environment, you saw how to navigate smart contracts and the basics of using Chainweaver developer tools, including the interactive Pact REPL.
+In [Set up a local development network](/build/pact/dev-network), you learned how to set up a local development environment and connect to it using Chainweaver. 
+Chainweaver provides a Module Explorer as part of its development environment.
```

Comment:
Since we don't actively develop chainweaver, and because you use the pact-cli REPL in this tutorial, it may be better to remove these references to chainweaver.